### PR TITLE
Revise and improve the documentation

### LINKED ISSUE #57: Documentation Revision
Revise the documentation to make the following changes:
- [x] In the `installation page` add a section about installation instructions for Archlinux users.
- [x] Update the `configuration page` to add and explain new options recently added.
- [x] Fix the `Go back to home` button in the documentation pages.
- [x] Update the `Why Websurfx?` section on the `faq page` to reflect the same changes as in #56.
- [x] Update the `theming page` to show the code about the `About Page` under the section `Themes > Custom`.

### PR DESCRIPTION
**Pull request by neon-mmd**
## What does this PR do?

This PR adds several changes and improvements to the documentation under the docs folder.

## Why is this change important?

This change is important to keep the documentation in sync with the current changes.

## Author's checklist

- [x] In the `installation page` add a section about installation instructions for Archlinux users.
- [x] Update the `configuration page` to add and explain new options recently added.
- [x] Fix the `Go back to home` button in the documentation pages.
- [x] Update the `Why Websurfx?` section on the `faq page` to reflect the same changes as in #56.
- [x] Update the `theming page` to show the code about the `About Page` under the section `Themes > Custom`.

## Related issues

Closes #57

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -1,30 +1,65 @@
...
+
+To get started with Websurfx, clone the repository, edit the config file which is located in the `websurfx` directory and install redis server by following the instructions located [here](https://redis.io/docs/getting-started/) and then build and run the websurfx server by running the following commands:
 
 ```shell
 git clone https://github.com/neon-mmd/websurfx.git
 cd websurfx
-cargo build
+cargo build -r
 redis-server --port 8082 &
 ./target/debug/websurfx
```
Comment (XFFXFF) [inline docs/installation.md:39]: You may have forgotten to update the path of the executable file, if we build in release mode, the path should be `./target/release/websurfx"

File: docs/installation.md [hunk 2]
```diff
@@ -1,30 +1,65 @@
...
 If you want to use the rolling/edge branch, run the following commands instead:
 
 ```shell
 git clone https://github.com/neon-mmd/websurfx.git
 cd websurfx
 git checkout rolling
-cargo build
+cargo build -r
 redis-server --port 8082 &
 ./target/debug/websurfx
```
Comment (XFFXFF) [inline docs/installation.md:56]: Same as the above

**Review by XFFXFF — CHANGES_REQUESTED** [review]
Very clear document, your writing is great!

File: docs/installation.md (hunk 2 above)
Comment (neon-mmd) [reply docs/installation.md:56]: Also this one as well I just overlooked it, I think next time I will have to be more careful :slightly_smiling_face: .

File: docs/installation.md (hunk 1 above)
Comment (neon-mmd) [reply docs/installation.md:39]: My bad, I didn't notice this at all smile but thanks for pointing it slightly_smiling_face .

**Review by XFFXFF — APPROVED** [review]

**Pull request merged**